docs/Contributing/API-for-contributors.md

```diff
@@ -2724,7 +2726,30 @@ If both `team_id` and `team_name` parameters are included, this endpoint will re
 
 #### Example
 
-`POST /api/v1/fleet/mdm/scripts/batch`
+`POST /api/v1/fleet/scripts/batch`
```

Comment:
```suggestion
`POST /api/v1/fleet/scripts/batch`
```

Is this API change part of this story or a separate one? I'm on board (I think we want to to be pulled out of `mdm`) just curious.

Reply:
I think this is a typo. In this example it's `...fleet/mdm/scripts`, but if you take a look above you'll see this endpoint and run script asynchronously endpoint don't include `/mdm`. @roperzh Could you confirm?

Probably I shouldn't add script updates in this PR it's confusing, but while I was working on software I realized we use "Uploaded" language instead of "Added", so I just added couple of copy changes to scripts section.

Reply:
it's a typo, thanks so much for fixing it!